doc/nrf/ug_thingy53.rst

```diff
@@ -302,53 +279,151 @@ See `Updating Nordic Thingy:53 firmware`_ for the detailed procedures on how to
 Thingy:53 application guide
 ***************************
 
-Thingy:53 does not have a built-in J-Link debug IC.
-Because of that, the samples and applications that are compatible with Thingy:53 by default use MCUboot bootloader with serial recovery support.
-Thingy:53-compatible applications can be updated using DFU functionality with either `nRF Connect Programmer`_ or nRF Programmer mobile application.
+The Nordic Thingy:53 does not have a built-in J-Link debug IC.
+Because of that, the samples and applications that are compatible with the Nordic Thingy:53 by default use MCUboot bootloader with serial recovery support.
+You can update applications compatible with the Nordic Thingy:53 using the DFU functionality with either `nRF Connect Programmer`_ or the `nRF Programmer`_ mobile application.
 
 Enabling DFU and MCUboot requires consistency in configuration of the samples and applications.
-For a sample or application to be compatible with Thingy:53, the application must comply with the following configuration:
+For a sample or application to be compatible with Thingy:53, the application must comply with the configurations described in the following sections.
+
+.. note::
+   To use your application with the Nordic Thingy:53 preprogrammed bootloader, you must only set the proper configuration for the Partition Manager and MCUboot.
+   Other mentioned features are nice to have, but they are not mandatory.
```

Comment:
Suggest rephrasing line 291 to: Configurations like USB logging and FOTA are not mandatory, but can be enabled if needed.

Reply:
Adapted to not mention specific features for increased compatibility and future-proofing.